src/h.ts

```diff
@@ -1,8 +1,9 @@
 import {vnode, VNode, VNodeData} from './vnode';
 export type VNodes = Array<VNode>;
-export type VNodeChildElement = VNode | string | number | undefined | null;
+export type VNodeChildElement = VNode | string | number | boolean | undefined | null;
 export type ArrayOrElement<T> = T | T[];
 export type VNodeChildren = ArrayOrElement<VNodeChildElement>
+export type FunctionComponent = (props: any, children?: VNode[]) => VNode;
```

Comment:
Can there be some type checking on `props`?

Reply:
fixed